src/styles/colors.css

```
@@ -12,16 +12,16 @@
-  --color-blue400: #5a70ff;
-  --color-blue500: #4459dd;
-  --color-blue600: #3244bb;
-  --color-blue700: #223299;
-  --color-blue800: #152277;
-  --color-blue900: #0c1555;
+  --color-blue50: #EBF8FF;
+  --color-blue100: #CAEBFC;
+  --color-blue200: #A1DFFF;
+  --color-blue300: #66C7FF;
+  --color-blue400: #33ADFF;
+  --color-blue500: #0088FF;
```

Stylelint (17.2.0)

[error] Expected "#0088FF" to be "#08F" (color-hex-length)

(color-hex-length)